attributes/default.rb

```diff
@@ -161,8 +169,8 @@
 default['ndb']['local_backup_dir']                    = "#{node['ndb']['root_dir']}/ndb/backups"
 default['ndb']['restore']['tarball']                  = ""
 default['ndb']['restore']['backup_id']                = ""
-default['ndb']['restore']['exclude_databases_meta']   = "glassfish_timers.EJB__TIMER__TBL"
-default['ndb']['restore']['exclude_databases_data']   = "glassfish_timers.EJB__TIMER__TBL,hopsworks.hosts,hopsworks.host_services"
+default['ndb']['restore']['exclude_databases_meta']   = "mysql.ndb_apply_status,glassfish_timers.EJB__TIMER__TBL"
```

Comment:
EJB__TIMER__TBL is not restored but then the replication tries to apply changes to tables in that database.